packages/main/src/plugin/docker/docker-compatibility.ts

```diff
@@ -69,14 +69,17 @@ export class DockerCompatibility {
   }
 
   protected getTypeFromServerInfo(
-    info: { OperatingSystem?: string },
+    info: { Name?: string; OperatingSystem?: string },
     podmanInfo?: unknown,
   ): DockerSocketServerInfoType {
     if (info.OperatingSystem === 'Docker Desktop') {
       return 'docker';
     } else if (info.OperatingSystem === 'podman' || podmanInfo) {
       // if podman info is available, then it is podman
       return 'podman';
+    } else if (info.Name?.startsWith('lima')) {
```

Comment:
what exactly `name` is this?
i mean, isn't this a user-defined name? If so - we can't rely on it, cause it may be anything

Reply:
It is the hostname.